src/spawn.coffee

```diff
@@ -30,12 +30,17 @@ procNotFoundError = (error, cmd) ->
   error.message = "Unable to find #{cmd}"
   error
 
-interpolatePort = (port) ->
-  (arg) ->
-    arg.replace '%port%', port
+processPorts = {}
+interpolatePorts = (args, processName, port) ->
+  processPorts["#{processName}.port"] = port
```

Comment:
I'd rather not use a full template language. Allowing the user to use all of that when we only need some simple replacements doesn't feel necessary. It also might lead to some strange uses of subprocess that we'd rather find a better way to support.